waltid-services/waltid-e2e-tests/src/test/kotlin/AuthApi.kt

```diff
@@ -192,7 +192,7 @@ class AuthApi(private val client: HttpClient) {
         )
         private val subjectJWKPrivateKey = PKIXUtils.javaPrivateKeyToJWKKey(subjectKeyPair.private)
 
-        private fun createX5CAccountRequest(key: JWKKey, cert: X509Certificate) = runBlocking {
+        private fun createX5CAccountRequest(key: JWKKey, cert: X509Certificate): AccountRequest = runBlocking {
```

Comment:
this feels like a change coming from https://github.com/walt-id/waltid-identity/pull/746